tests/test_authentication.py

```diff
@@ -140,13 +139,17 @@ def test_unknown_creds(self):
                               '^access denied: CredentialsLookupError: ')
 
     def test_alternative_credential_lookup(self):
+        # Temporarily use alternate auth
+        old_auth = self.auth
+        self.auth = AlternateAuthentication()
+
         sender = self._sender(credentials=ALTERNATIVE_CREDS)
         req = self._request(sender)
-        lookup_path = "%s.alternative_lookup" % __name__
-        with self.settings(HAWK_CREDENTIALS_LOOKUP=lookup_path):
```

Comment:
This existing test should not be deleted because the `HAWK_CREDENTIALS_LOOKUP` setting is still supported. The new test you added is necessary but it shouldn't replace the old one.